src/main/java/com/edmunds/tools/databricks/maven/WorkspaceToolMojo.java

```diff
@@ -40,6 +40,7 @@
 /**
  * Interacts with the databricks workspace api.
  */
+@Deprecated
```

Comment:
It might be useful to include a note in the deprecation text, such as "please use the export-workspace mojo instead".

Reply:
good point!